wiki/Tournaments/TWC/2023/en.md

```diff
@@ -0,0 +1,202 @@
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee
+   - On subsequent occurrences:
+     - For a pick timer: a random map will be chosen from the mappool using `!roll X`, where X is the number of beatmaps that were neither picked nor banned, excluding the tiebreaker.
+     - For a ready timer: the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+     - Repeat offenders may receive further sanctions from the tournament managers.
+6. Each team will receive one "tactical timeout" of 2 minutes, to be used as extra time to pick, ban, or ready for a beatmap. *The tactical timeout is optional.*
+   - The tactical timeout may be called on a tiebreaker, provided that both teams' captains agree to it.
+
+### Mappool instructions
+
+1. Every stage will have its own mappool.
+2. Each mappool consists of 5 brackets: [NoMod](/wiki/Game_modifier#nomod), [Hidden](/wiki/Game_modifier/Hidden), [Hard Rock](/wiki/Game_modifier/Hard_Rock), [Double Time](/wiki/Game_modifier/Double_Time) and [FreeMod](/wiki/Game_modifier#freemod).
+3. The mappool sizes are as follows:
+   - Qualifiers: 10 beatmaps
+   - Round of 32 and Round of 16: 14 beatmaps
+   - Quarterfinals and Semifinals: 15 beatmaps
+   - Finals and Grand Finals: 16 beatmaps
+4. Each mappool has one tiebreaker, except for the Qualifiers.
+5. The NoMod bracket will be played with the [No Fail](/wiki/Game_modifier/No_Fail) mod activated.
+6. The [Hidden](/wiki/Game_modifier/Hidden), [Hard Rock](/wiki/Game_modifier/Hard_Rock) and [Double Time](/wiki/Game_modifier/Double_Time) brackets will be played with the respective mods activated along with the No Fail mod.
+7. The [FreeMod](/wiki/Game_modifier#freemod) bracket will have Free Mods activated. The valid mod combinations for each player are:
+   - [NoMod](/wiki/Game_modifier#nomod) – no mods selected
+   - Hidden
+   - Hard Rock
+   - Hidden + Hard Rock
+8. For FreeMod, each team is **required** to have both Hidden AND Hard Rock selected among its players. A player may shoulder all the mods for their team (e.g. one player selects Hidden + Hard Rock, and the remaining players select no mods (NoMod)), or it may be split among all members (e.g. the first player selects Hidden, the second selects Hard Rock, and the third selects no mods (No Mod)). Teams may "overmod", that is, select more mods than what is required.
+9. Failed scores will be counted in their integrity. To that end, the [No Fail](/wiki/Game_modifier/No_Fail) mod will be enforced on every beatmap.
+   - Teams that purposefully disregard this instruction and team fail as a result will lose said game.
```

Comment:
in what way could you purposefully disregard this... ? i don't think i understand this rule